sandbox/Cargo.toml

```diff
@@ -5,11 +5,11 @@ publish = false
 authors = ["The Exonum Team <exonum@bitfury.com>"]
 
 [dependencies]
-exonum = {version="0.5", path="../exonum"}
+exonum = { version = "0.5", path = "../exonum" }
 
 log = "0.4.1"
 env_logger = "0.5.3"
-clap = {version="2.25.0", features=["unstable"]}
+clap = { version = "2.25.0", features = ["unstable"] }
```

Comment:
Do we need to use `unstable` features in `clap`?

Reply:
Honestly, I don't know (yet). I'm going to check it a little bit later.
Anyway this isn't a new change. :smile: